helm/hpcc/templates/_helpers.tpl

```diff
@@ -121,20 +151,24 @@ storage:
     labels:
     - data
   {{- end }}
-{{ toYaml (unset (unset (deepCopy $plane) "name") "pvc")| indent 4 }}
+{{ toYaml (omit $plane "name" "pvc" "storageClass" "storageSize") | indent 4 }}
  {{- end }}
 {{- end }}
 {{- /* Add implicit planes if data or spill storage plane not specified*/ -}}
 {{- if not $dataStorage.plane }}
+{{- if not (include "hpcc.hasPlaneForLabel" (dict "root" $ "label" "data")) }}
   - name: hpcc-data-plane
     labels:
     - data
     prefix: {{ .Values.global.defaultDataPath | default "/var/lib/HPCCSystems/hpcc-data" | quote }}
 {{- end }}
+{{- end }}
 {{- if not $spillStorage.plane }}
+{{- if not (include "hpcc.hasPlaneForLabel" (dict "root" $ "label" "spill")) }}
```

Comment:
Does this work? hasPlaneForLabel will return empty string or "true" rather than a boolean, but perhaps that's ok for an if ...